test/gov/verify_agreements.sh

```diff
@@ -9,8 +9,43 @@ echo -e "${PREFIX} starting"
 function verifyAgreements {
         # Wait until there are agreements
         TARGET_NUM_AG=1
+        MUL_NUM_AG_1=0
+        MUL_NUM_AG_2=0
+        MUL_NUM_AG_3=0
+        MUL_NUM_AG_4=0
         if [ "${HZN_REG_TEST}" != "1" ]; then
             if [ "${PATTERN}" == "sall" ]; then
+              case ${MUL_AGENTS} in
```

Comment:
there is way too much duplicated code here, we need to find a better way to deal with this.